src/app/shared/text-overflow/text-overflow-webkit-line-clamp.component.css

```diff
@@ -0,0 +1,10 @@
+div {
```

Comment:
This is what we were using previously. There was some discussion of just using the webkit line clamp, so I kept it around. Considering the drawbacks on the fade and cross browser ellipses techniques, figured we might eventually want this in the style guide in cases where the other two just aren't the right choice.